addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/timing/TimingUtils.java

```diff
@@ -17,11 +17,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.zaproxy.zap.extension.ascanrules.timing;
+package org.zaproxy.addon.commonlib.timing;
 
 import java.io.IOException;
 
-/** Utility class to host time-based blind detection algorithms. */
+/**
+ * Utility class to host time-based blind detection algorithms.
+ *
+ * @since 1.15.0
```

Comment:
This PR should be rebased and adjusted accordingly. commonlib 1.18 was released last week.

The CHANGELOGs in the ascan rules add-ons likely need to be re-jigged due to releases as well.